samples/nrf9160/modem_shell/overlay-ppp.conf

```diff
@@ -14,6 +14,8 @@ CONFIG_MOSH_CURL=n
 CONFIG_MOSH_LINK=y
 CONFIG_MOSH_GNSS=n
 CONFIG_MOSH_SMS=n
+CONFIG_MOSH_LOCATION=n
+CONFIG_MOSH_REST=n
```

Comment:
Just adding a note here that I had to disable something to avoid SRAM running out in PPP configuration:
```
10:05:58  INFO    - 47/72 nrf9160dk_nrf9160_ns      nrf9160/modem_shell/samples.nrf9160.modem_shell.ppp FAILED Build failure (build)
...
10:05:58  /opt/gnuarmemb/gcc-arm-none-eabi-9-2019-q4-major/bin/../lib/gcc/arm-none-eabi/9.2.1/../../../../arm-none-eabi/bin/ld.bfd: zephyr/zephyr_prebuilt.elf section `noinit' will not fit in region `SRAM'
10:05:58  /opt/gnuarmemb/gcc-arm-none-eabi-9-2019-q4-major/bin/../lib/gcc/arm-none-eabi/9.2.1/../../../../arm-none-eabi/bin/ld.bfd: region `SRAM' overflowed by 8528 bytes
```